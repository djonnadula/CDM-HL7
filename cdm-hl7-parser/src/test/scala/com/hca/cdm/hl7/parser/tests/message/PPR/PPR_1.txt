MSH|^~\&|COCQA1A|COCQA1A|OV|OV|218799871512||PPR^PC1|IP_COQ1A_PPR_EPSQ1A.21654.4561|T|2.5||KYA^04446|ER|NE
PID|1||K125412312^^^COCQA1A^MRN|PO132541243|SPARLKER^VENDDIAHGRAM^R||19872737|F
PRB|DE|987954564987^QADGF1245123^DRBRI|285744009^Formed hallucinations of people^SNOMED|PO132541243-5|||987954564987^QADGF1245123^DRBRI|||Deleted||Deleted
NTE|1|CM|Reason for removal: Testing removal of formed hallucinations