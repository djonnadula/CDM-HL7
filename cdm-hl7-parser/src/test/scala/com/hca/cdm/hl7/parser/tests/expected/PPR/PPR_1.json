{
  "0000.COMN": {
    "sending_facility": "COCQA1A",
    "message_control_id": "IP_COQ1A_PPR_EPSQ1A.21654.4561",
    "medical_record_num": "K125412312",
    "medical_record_urn": "PO132541243",
    "patient_account_num": ""
  },
  "0001.MSH": {
    "field_separator": "|",
    "encoding_characters": "^~\\&",
    "sending_application": "COCQA1A",
    "sending_facility": "COCQA1A",
    "receiving_application": "OV",
    "receiving_facility": "OV",
    "date_time_of_message": "218799871512",
    "security": null,
    "message_type": {
      "message_code": "PPR",
      "trigger_event": "PC1"
    },
    "message_control_id": "IP_COQ1A_PPR_EPSQ1A.21654.4561",
    "processing_id": "T",
    "version_id": "2.5",
    "sequence_num": null,
    "continuation_pointer": {
      "network_id": "KYA",
      "unit_number": "04446"
    },
    "accept_acknowledgement_type": "ER",
    "application_acknowledgement_type": "NE"
  },
  "0002.PID": {
    "set_id": "1",
    "patient_id": null,
    "patient_id_list": {
      "id_num": "K125412312",
      "id_check_digit": null,
      "check_digit_scheme": null,
      "assigning_authority": "COCQA1A",
      "id_type_code": "MRN"
    },
    "alternate_patient_id": "PO132541243",
    "patient_name": {
      "family_name": "SPARLKER",
      "given_name": "VENDDIAHGRAM",
      "second_and_further_given_names_or_initials_thereof": "R"
    },
    "mother_maiden_name": null,
    "date_time_of_birth": "19872737",
    "administrative_sex": "F"
  },
  "0003.PRB": {
    "action_code": "DE",
    "action_date_time": {
      "date_time": "987954564987",
      "person_id": "QADGF1245123",
      "provider_id": "DRBRI"
    },
    "problem_id": {
      "id": "285744009",
      "text": "Formed hallucinations of people",
      "name_of_coding_system": "SNOMED"
    },
    "problem_instance_id": "PO132541243-5",
    "episode_of_care_id": null,
    "problem_list_priority": null,
    "problem_established_date_time": {
      "date_time": "987954564987",
      "person_id": "QADGF1245123",
      "provider_id": "DRBRI"
    },
    "anticipated_problem_resolution_date_time": null,
    "actual_problem_resolution_date_time": null,
    "problem_classification": "Deleted",
    "problem_management_discipline": null,
    "problem_persistence": "Deleted"
  },
  "0004.NTE": {
    "set_id": "1",
    "source_of_comment": "CM",
    "comment": "Reason for removal: Testing removal of formed hallucinations"
  }
}